src/main/java/org/openlmis/buq/service/referencedata/RightReferenceDataService.java

```diff
@@ -49,4 +49,9 @@ public RightDto findRight(String name) {
     return rights.isEmpty() ? null : rights.get(0);
   }
 
+  public List<RightDto> findRights(List<String> names) {
+    List<RightDto> rights = findAll("search", RequestParameters.init().set("name", names));
+    return rights.isEmpty() ? null : rights;
+  }
```

Comment:
I think this can be removed, because unfortunately it was not working, so I used `findRight` in a loop not to waste time of making it work, because it won't be that costly even if used in the loop.